sqlvm-provisioning-csp/SQLServerVMDeployment.json

```diff
@@ -0,0 +1,238 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vmName": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "SQL IaaS VM machine name"
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "SQL IaaS VM size. Use Azure VM size name from MSDN"
+            }
+        },
+        "sqlImageOffer": {
+            "type": "string",
+            "defaultValue": "SQL2014-WS2012R2",
+            "allowedValues": [
+                "SQL2014-WS2012R2"
+            ],
+            "metadata": {
+                "description": "SQL Server Gallery Image Offer"
+            }
+        },
+        "sqlImageSku": {
+            "type": "string",
+            "defaultValue": "Standard",
+            "allowedValues": [
+                "Enterprise",
+                "Standard",
+                "Web"
+            ],
+            "metadata": {
+                "description": "SQL Server Gallery Image SKU"
+            }
+        },
+        "sqlImageVersion": {
+            "type": "string",
+            "defaultValue": "latest",
+            "metadata": {
+                "description": "SQL Server Gallery Image Published Version"
+            }
+        },
+        "vmLocation": {
```

Comment:
Why are you asking us to remove default values. Can you please provide a reason.